### ISSUE TITLE
Add capability to archive/unarchive models from a model registry programmatically

### ISSUE THREAD

**Issue by hariishaa** [allegroai/clearml#1096]
## Proposal Summary
Capability to archive/unarchive models from a model registry programmatically.

## Motivation
I would like to manage states of models in model registry from unpublished to published and then from published to archived and do all this actions from code. Currently only a function publish() is available in Model class, but function archive() is absent.

**Comment by eugen-ajechiloae-clearml** [comment]
Hi @hariishaa ! Thank you for the suggestion. We will soon add this functionality.

**Comment by hariishaa** [comment]
@pollfly this is good news! Thank you!
But unfortunately I noticed that I am also missing the 'archived' attribute to determine the status of a model in a registry (whether it is archived or not).

**Comment by ainoam** [comment]
@hariishaa the archive status of a model is noted through the 'archived' system tag. Does this help?

**Comment by hariishaa** [comment]
> @hariishaa the archive status of a model is noted through the 'archived' system tag. Does this help?

Yeah, that's a nice workaround, but it's not obvious from the docs.
I am sure that it would be better to have 'archived' attribute for the models.
